cockroachcloud/use-managed-service-backups.md

```diff
@@ -125,6 +124,23 @@ Additional ways to restore data:
 - [Back up a self-hosted CockroachDB cluster and restore into a {{ site.data.products.db }} cluster](#back-up-a-self-hosted-cockroachdb-cluster-and-restore-into-a-cockroachdb-cloud-cluster)
 - [Back up and restore data manually](take-and-restore-customer-owned-backups.html)
 
+### Restore a cluster
+
+To restore a cluster:
+
+1. Find the cluster backup on the **Backups** page.
+1. Click **Restore** for the cluster you want to restore.
+
+    The **Restore cluster** module displays with backup details.
+
+1. Click **Continue**
+    
+    The restore will completely erase all data in the cluster. All cluster data will be replaced with the data from the backup.
```

Comment:
Do we want to add this to a warning note?

Also, where this is placed, under 3, it seems like the data is erased at this point — is that true? 

I think adding this to a warning box will help with this second point ^

Reply:
Maybe where you have it makes sense then, if a reader is just about to, or has just clicked the continue step!

Thanks for the screen grab! Yeah, adding that into a warning callout in the docs will be nice symmetry here. (As Ben noted.)